### PR TITLE
#167869451 create trips using postgres db

### PR DESCRIPTION
**Pull request by WinnersProx**
#### What does this PR do?
- implement persistent trips creation using Postgres
#### Description of Task to be completed?
- link create the trip endpoint with Postgres DB
#### How should this be manually tested?
- to test this just clone the repo and read the documentation
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/167869451

### CODE REVIEW

File: server/v1/routes/index.js [hunk 1]
```diff
@@ -2,6 +2,8 @@ import express from 'express';
 import authRouter from './auth';
 import tripsRouter from './trips';
 import bookingsRouter from './bookings';
+import authenticationV1 from '../middlewares/authentication';
+
 const router = express.Router()
 const docsUrl = 'https://way-farer-app-rest.herokuapp.com/api/v1/api-docs/';
```
Comment (oesukam) [inline server/v1/routes/index.js:8]: The URL should go in the `.env` file

File: server/v2/helpers/user_helper.js [hunk 2]
```diff
@@ -1,17 +1,21 @@
...
   return bcrypt.compareSync(plain, encrypted) ? true : false;
 }
-const authenticateUser = ({id,email,first_name}) => {
+const authenticateUser = ({id,email,first_name, is_admin}) => {
+  const { SECRET } = process.env;
   return jwt.sign({
     id,
     email,
-    first_name
+    first_name,
```
Comment (oesukam) [inline server/v2/helpers/user_helper.js:15]: @WinnersProx You only need the email in the object, you should remove the id, first_name and, last_name and retrieve the user's information when validation the token

File: server/v2/routes/index.js [hunk 3]
```diff
@@ -3,8 +3,6 @@ import authRouter from './auth';
 import tripsRouter from './trips';
 import bookingsRouter from './bookings';
 import apiV2Authentication from '../middlewares/authentication.js';
-const app = express();
-app.use(apiV2Authentication.initialize());
 const router = express.Router();
 const docsUrl = 'https://way-farer-app-rest.herokuapp.com/api/v1/api-docs/';
```
Comment (oesukam) [inline server/v2/routes/index.js:7]: Needs to go in `.env`

**Review by oesukam — CHANGES_REQUESTED** [review]

**Comment by oesukam** [comment]
@WinnersProx Your Travis is failing

**Comment by WinnersProx** [comment]
> @WinnersProx Your Travis is failing

it's failing because of some features I'm adding I will shortly fix it

**Pull request merged**